hiplan/base/app.py

```diff
@@ -0,0 +1,31 @@
+class App:
+    __instance = None
+    def __init__(self):
+        if App.__instance is not None:
+            raise Exception("App was alredy instantiated")
+
+        App.__instance = self
+        
+        self.boards: List[Board] = []
```

Comment:
Aici o sa ai nevoie de `List` (`typing`) si de `Board` (`hiplan`). Rezolva restul problemelor de genul din toate fisierele. Importa doar ce ai nevoie

Reply:
Am incercat sa rulez si nu mergea pt ca nu recunostea Board in app, asa ca am incercat sa import ce imi trebuie, dar spune ModuleNotFoundError. Dar base are
pycache
│   ├── app.cpython-38.pyc
│   ├── board.cpython-38.pyc
│   └── member.cpython-38.pyc

Reply:
Am modificat importurile, dar nu imi recunoaste hiplan ca modul.

Reply:
> Am modificat importurile, dar nu imi recunoaste hiplan ca modul.

`pip3 install -e .` sau `python3 setup.py develop` in root-ul proiectului

Reply:
Tot nu merge:
 ModuleNotFoundError: No module named 'hiplan.base.task.py'; 'hiplan.base.task' is not a package